hyperframe/flags.py

```diff
@@ -7,11 +7,16 @@
 """
 import collections
 
+try:
+    import collections.abc as collections_abc
+except ImportError:
```

Comment:
The tests are failing because they enforce 100% coverage, and only one of these branches gets executed in any particular version of Python.

If you add a comment to this line:

```suggestion
except ImportError:  # Python 2.7
```

and then add a `.coveragerc` file to the root of the repo, I think that should satisfy the coverage checks:

```
[report]
fail_under = 100
show_missing = True
exclude_lines =
    .*:.* # Python \d.*
```

I'll open a PR to fix the flake8 issues separately.

Reply:
I'm already on it... PR incoming in 10min or so...